### ISSUE TITLE
How do you save mode weights?

### ISSUE THREAD

**Issue by LinkHS** [bonlime/keras-deeplab-v3-plus#75]
Hi Bonlime,
Everything is fine when I use model.load_weights() to load your pretrained model.

However, weights can't be loaded correctly from the model.h5, which I saved by `model.save()` or `model.save_weights()`. I printed each layer and their weights all become `nan`.

I've built and test a simple model as below, it seems that `    model.add(BatchNormalization(epsilon=1e-3))` caused the problem.
> If I comment out `model.add(BatchNormalization(epsilon=1e-3))` then no problem. 

```
def SimpleModel():
    import keras
    model = keras.models.Sequential()

    model.add(Conv2D(6, (1, 1), input_shape=(512, 512, 3)))
    model.add(BatchNormalization(epsilon=1e-3))
    model.add(Activation('relu'))   

    model.add(Conv2D(21, (1, 1)))

    return model
```

I am wondering that you also used BatchNormalization but the model.h5 is OK to be loaded. Could you please tell me how do you save the model?

Thanks!

Austin
